README.md

```diff
@@ -5,9 +5,9 @@ Each directory contains a homework problem, four in total. Please complete one--
 more--of the four before submitting. Make sure to read the README in each directory,
 it will give you important information on completing each problem.
 
-We're on the honor system here. Don't get help from, or give help to, another
-person or a resource on the internet. Do the work yourself. It's okay if you
-need to consult language docs or the MDN or similar.
+We're on the honor system here, and the goal is to show what you're capable of.
+Feel free to use any resources you like in solving the problems
+but don't ask others for help or make use of solutions you may find out there on the net.
```

Comment:
Hoe about "pre-existing solutions"? Finding a post about how to add an event listener for a remove button is okay, but finding someone else's homework is not.